packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html

```diff
@@ -1,7 +1,24 @@
 <h1
+    class="mb-1"
     data-translate="pci_projects_project_storages_containers_add_title"
-    data-ng-if="!$ctrl.archive"
+    data-ng-if-start="!$ctrl.archive"
```

Comment:
i think it's unusual in the manager to use this angular syntax, for consistency if you want to avoid having the same test multiple time maybe you can wrap both h1 & h2 in another html tag and add the condition on this one

in my opinion i see one issue with the *-start *-end syntax : it makes the code difficult to read as you have to keep track of the starting and ending point (conterintuitive with the native html tree-like syntax)

Reply:
I see. The counter part is we have extra "useless" markup but I guess it's worth it :+1: